client/src/components/update.jsx

```diff
@@ -46,10 +47,20 @@ const Update = ({ update }) => {
 
   let desc = null;
   let backersOnlyDesc = null;
-  if (!backersOnly) {
+  const shortDescription = helpers.cutOffDesc(description);
+  if (!backersOnly && shortDescription.length === description.length) {
     desc = (
       <p>
-        {helpers.cutOffDesc(description)}
+        {description}
+      </p>
+    );
+  } else if (!backersOnly && shortDescription.length !== description.length) {
+    desc = (
+      <p>
+        {shortDescription}
+        <span style={{ textDecoration: 'underline' }}>
+          Read more
```

Comment:
@zcoursey22 you can either use the <u></u> element here, or a styled-component. Inline css is not best practice.